barista-kafka/src/main/resources/META-INF/microprofile-config.properties

```diff
@@ -1,8 +1,6 @@
 barista.name=julie
 
-mp.messaging.connector.liberty-kafka.bootstrap.servers=localhost:9092
-
-#mp.messaging.connector.liberty-kafka.bootstrap.servers=kafka-cluster-kafka-bootstrap:9092
+mp.messaging.connector.liberty-kafka.bootstrap.servers=kafka-cluster-kafka-bootstrap:9092
```

Comment:
Is this a more appropriate default than localhost? I think this value is only appropriate if the Strimzi operator creates a Kafka named 'kafka-cluster'.

Reply:
Good point.